src/AppInstallerCLIE2ETests/GroupPolicy.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+
+namespace AppInstallerCLIE2ETests
+{
+    /// <summary>
+    /// Tests for enforcement of Group Policy.
+    /// Behavior is better tested in the unit tests; these tests mostly ensure match between the code and the definition.
+    /// </summary>
+    public class GroupPolicy : BaseCommand
+    {
+        [SetUp]
+        public void Setup()
+        {
+            InitializeAllFeatures(false);
+            GroupPolicyHelper.DeleteExistingPolicies();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            InitializeAllFeatures(false);
+            GroupPolicyHelper.DeleteExistingPolicies();

```

Comment:
Post-approval comment: This could cause trouble if the user actually has policies that should be in place.  I don't know of a better way to do this other than product code changes though.

Reply:
The only trouble I think this could cause is leaving the policy not as it should be, or allowing to bypass the policy.

The configuration for registry-based policies is stored in a [Registry.pol file](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/Policy/registry-policy-file-format) in the system. I believe that if we modify the registry directly, that policy will at some point overwrite anything we change (but I'm not clear on the details), so it will all eventually be as it should.

And anybody running these tests could already have their version of `wingetdev` that ignores policy, so I don't think we need to worry about this opening the option to bypass it.